citadel/install.md

```diff
@@ -52,3 +40,28 @@ collection assures that all libraries are compatible and can be used together.
 |   ign-tools        |       1.x     |
 |   ign-transport    |       8.x     |
 |   sdformat         |       9.x     |
+
+## Supported platforms
+
+Citadel is [supported](/docs/all/releases) on the platforms below.
+
+These are the officially supported platforms:
+
+* Ubuntu Bionic on amd64/i386
+* Ubuntu Focal on amd64
+
+All other platforms on the table are supported at best-effort.
+
+Operating System | Architecture | Packaging | CI | Functionality
+---------------- | ------------ | --------- | -- | -------------
+Ubuntu Bionic    | amd64        | ✅ Yes    | ✅ Yes, every pull request | ✅ All
+.                | i386         | ✅ Yes    | No | ✅ All
+.                | arm64        | ❓ Maybe  | No | Most low-level libraries known to work
+.                | armhf        | ❓ Maybe  | No | Most low-level libraries known to work, DART physics engine not available
+Ubuntu Focal     | amd64        | ✅ Yes    | ✅ Yes, every pull request | ✅ All
+.                | arm64        | ❓ Maybe  | No | Most low-level libraries known to work
+.                | armhf        | ❓ Maybe  | No | Most low-level libraries known to work, DART physics engine not available
+Debian Buster    | amd64, i386, arm64, armhf | ❓ Maybe | No | Several libraries known to work
+MacOS Mojave     | -            | ✅ Yes    | ✅ Yes, every pull request | Ignition only works in headless mode (GUI does not render; instead of using `ign gazebo fuel.sdf` command, use `ign gazebo -s fuel.sdf` to start the server only).
```

Comment:
we don't have any more CI machines running macOS Mojave

Reply:
I'm trying to figure out what platforms we run CI for and which we build bottles for each Igntiion version. Can you help me out?

I think this README may be out of date: https://github.com/osrf/homebrew-simulation/#jenkins-implementation-details

Reply:
I've updated the README as part of https://github.com/osrf/homebrew-simulation/pull/1695